src/main/java/org/cactoos/func/CallableOf.java

```diff
@@ -66,39 +58,42 @@
      * @since 0.32
      */
     public CallableOf(final Runnable runnable, final T result) {
-        this(new FuncOf<>(runnable, result));
+        this(() -> {
+            runnable.run();
+            return result;
+        });
     }
 
     /**
      * Ctor.
      * @param proc Encapsulated proc
+     * @param ipt Input
      * @param result Result to return
      * @since 0.32
```

Comment:
Constructor signature changed, so I think it should be considered a new constructor, and so the `@since` tag should updated.